src/SimpleRADOSStriper.h

```diff
@@ -59,7 +59,7 @@ class [[gnu::visibility("default")]] SimpleRADOSStriper
   int stat(uint64_t* size);
   ssize_t write(const void* data, size_t len, uint64_t off);
   ssize_t read(void* data, size_t len, uint64_t off);
-  int truncate(size_t size);
+  int truncate(uint64_t size);
```

Comment:
related?

Reply:
The issue is that clang on Darwin doesn't think that `size_t` and `uint64_t` are synonyms, and they shouldn't be. The code, however, has been quite loose around this matter. It could go either way - i.e. could have truncated the argument to size_t at the call site, but decided to update the method here.

Reply:
What I meant was this: size_t is (on 64b systems) of size 8, so there is no real binary diff between the two.
But the question was - why fix it in this specific case as part of the PR?
Anyway - I'm OK with it, but I'd rather someone who handled this part of the code to approve.

Reply:
The only reason was that the compilation failed unless this was addressed

Reply:
I believe it's reasonable to have this as `uint64_t` as in this case the size refers to the file size. We can see above that this is similar to `uint64_t off` in `read` and `write`, and to `uint64_t *size` in `stat`

Reply:
Yeah, the `uint64_t*` in `stat()` is a good point.